docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst

```diff
@@ -19,37 +19,27 @@ or more loose stools in a 24-hour period." (p. 88 of
 Modeling Diarrheal Diseases in GBD 2017
 ---------------------------------------
 
+The GBD diarrheal diseases model follows a standard GBD framework, including a cause of death (CoD) model and
+a nonfatal model.
 
+The CoD model estimates the cause-specific mortality rate (CSMR) within the total population, and a cause fraction.
+These estimates are based on vital registration and autopsy data.
```

Comment:
and _verbal_ autopsy data?

Reply:
Yes! Verbal autopsy; thanks for the catch